test/plugin.test.js

```diff
@@ -108,6 +108,58 @@ QUnit.test('no progress for 45 seconds is an error', function(assert) {
   assert.strictEqual(this.player.error().type, 'PLAYER_ERR_TIMEOUT');
 });
 
+QUnit.test('the plugin cleans up after its previous incarnation when called again',
```

Comment:
The timeout has a check to make sure `player.error()` doesn't return anything, so even if there are multiple monitors, only one error event will be fired, so we need to check that the listeners are actually being cleaned up